File: pset6/mario.py
import cs50

def main():
    while(True):
        print("Height: ", end="")
        height = cs50.get_int()
        if height >= 0 and height <= 23:
            break
    for i in range(height):
        blanks = height-(i+1)
        for j in range(blanks):
            print(" ", end="")
        for k in range(i+1):
            print("#",end="")
        print("  ", end ="")
        for l in range (i+1):
            print("#",end="")
        print("")




def get_positive_int():
    while True:
        n = cs50.get_int()
        if n >= 0 and n <= 23:
            break
    return n


if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main (void)
{
   int height=0;

   do
   {
       printf("Height:\n");
       height=get_int();
    }
    while(height<0 || height>=24);

    for(int i=0; i<height;i++)
    {
        int blanks=height-(i+1);

        for (int j=0;j<blanks;j++)
        {
            printf(" ");
        }

        for(int k=0;k<=i;k++)
        {
        printf("#");
        }

        printf("  ");

        for(int k = 0; k <= i; k++)
		{
			printf("#");
		}

      printf("\n");
    }
}


